### DISCUSSION TITLE
Problem in waveguide length measurement

### DISCUSSION THREAD

**Discussion by diteihub** [SiEPIC/SiEPIC-Tools#169] [Q&A]
I am facing an issue while trying to do waveguide length measurement, a picture is attached below to show the exact problem.
 If anyone has some idea to solve this I would greatly appreciate it. The version of the software is "K-layout 0.27.9.
Thank you

![wg_length_measurement_issue](https://user-images.githubusercontent.com/106652525/171843333-facc3ed1-e22b-45c3-9419-49035a97d482.png)

**Comment by jevillegasd** [comment]
hey did you figure it out. Could you share your file?
The measurement tool works well on my side (I am using SiEPIC version 0.3.91), maybe updating yours might help.

**Comment by lukasc-ubc** [comment]
For now please 1) update siepic tools, 2) zoom in to the start of the waveguide and read the number there. 
>
